### DISCUSSION TITLE
Usage of omega and sigma methods

### DISCUSSION THREAD

**Discussion by hcars** [networkx/networkx#4612] [Q&A]
For some reason these methods always seem to be in an infinite loop, is this a user error?

**Comment by rossbar** [comment]
Can you provide some info about the size of graphs in your use-case? A minimum reproducing example would be very helpful in IDing whether there's a problem and if so where it might be.

**Comment by dschult** [comment]
Try this (it took 1 minute on my machine):
```python
>>> G=nx.karate_club_graph()
>>> nx.omega(G)
0.35410806809770545
```
Keep in mind that this is a pretty small graph (34 nodes and 78 edges). 
If you look at the formula for these functions you will see that the amount
of computation required gets very large as the graph grows.
